CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # RME Changelog
 
+
+## Prochaine Release
+
+## Release (1.1.0)
+
```

Comment:
Maybe split into "Features" and "Bugfixes"

Reply:
Done